easybuild/easyconfigs/l/libcroco/libcroco-0.6.13-GCCcore-10.2.0.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'ConfigureMake'
+
+name = 'libcroco'
```

Comment:
Hm, well, this has a version at GCC level already, though, I suppose moving it to core is reasonable (not sure why it wasn't from the start)